cloud/deploys/staging/README.md

```diff
@@ -47,7 +48,7 @@ export ARM_ACCESS_KEY=$ACCOUNT_KEY
 
 ## Running terraform with this config  
 Since we are using a shared backend you'll need to run init specifying that 
-within the `cloud/azure` directory.
+within the `cloud/deploys/staging` directory.
```

Comment:
suggestion (blocking): I'd recommend renaming the path to `cloud/deploys/staging_azure/` since we're likely going to have a GCP staging env soon as well